### PR TITLE
[CQT-31] Add spin appendix

### PR DESCRIPTION
**Pull request by oschusler**

### CODE REVIEW

File: docker-compose.yaml [hunk 1]
```diff
@@ -1,5 +1,3 @@
-version: "3"
-
 services:
   mkdocs_qne:
```
Comment (elenbaasc) [inline docker-compose.yaml:2]: Btw, can we change this name? This seems to refer to the Quantum Network Explorer, right?

File: requirements.txt [hunk 2]
```diff
@@ -3,6 +3,6 @@ mkdocs-exclude==1.0.2
 mkdocs-material==9.5.22
 mkdocs-glightbox==0.4.0
 mkdocs-material-extensions==1.3.1
-mkdocs-build-plantuml-plugin==1.11.0
+mkdocs-build-plantuml-plugin==1.9.0
```
Comment (elenbaasc) [inline requirements.txt:6]: Why is this downgraded?

Comment (oschusler) [reply requirements.txt:6]: Version `1.10.0` has been yanked and version `1.11.0` is python 3.12 only. This container only runs on python 3.11.

File: docker-compose.yaml (hunk 1 above)
Comment (oschusler) [reply docker-compose.yaml:2]: This is indeed based on that project

**Review by elenbaasc — APPROVED** [review]

**Pull request merged**